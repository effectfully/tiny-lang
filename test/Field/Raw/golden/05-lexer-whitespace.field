for

i
	=
0to
100
do assert T; end



;



assert T;




