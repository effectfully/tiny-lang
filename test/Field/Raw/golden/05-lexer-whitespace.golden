Program (Statements [EFor (Var "i") 0 100 (Statements [EAssert (EConst (Some (UniConst Bool True)))]),EAssert (EConst (Some (UniConst Bool True)))])